modules/linear_boltzmann_solvers/lbs_solver/lbs_solver.cc

```diff
@@ -462,6 +462,9 @@ LBSSolver::OptionsBlock()
     "verbose_inner_iterations", true, "Flag to control verbosity of inner iterations.");
   params.AddOptionalParameter(
     "verbose_outer_iterations", true, "Flag to control verbosity of across-groupset iterations.");
+  params.AddOptionalParameter(
```

Comment:
Maybe `max` instead of `maximum` for smaller input parameter names?

Reply:
Fixed.